test/common/stats/allocator_impl_test.cc

```diff
@@ -524,6 +524,58 @@ TEST_F(AllocatorImplTest, ForEachSinkedGauge) {
   EXPECT_EQ(num_iterations, 0);
 }
 
+TEST_F(AllocatorImplTest, ForEachSinkedGaugeHidden) {
+  std::vector<GaugeSharedPtr> sinked_gauges;
+  std::vector<GaugeSharedPtr> unsinked_gauges;
```

Comment:
nit: you don't need vectors if you're only constructing one element.

Reply:
ditto next test point.

Reply:
Done